app/core/wallet.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
```

Comment:
introduced a core wallet call

Reply:
@morrieinmaas thanks for your changes - look how much more lightweight this call is!